### ISSUE TITLE
req:12 failed due to multiple serviceTypeVersion parameter in response.

### ISSUE THREAD

**Issue by keshav-nangare** [opengeospatial/ets-wcs20#19]
Failed tests:

Service URL: 
http://arcgis-ogcoregon-393577611.us-west-2.elb.amazonaws.com/arcgis/services/wcs201/GeoDataServer/WCSServer?SERVICE=WCS&VERSION=2.0.1&REQUEST=GetCapabilities

```

req12: For all WCS request types, the request version parameter shall have a fixed value of '2.0.1'.
Result: Failed.
[FAILURE] The version in the wcs:Capabilities result should be "2.0.1".
[MESSAGE] The server does not support the gml coverage.

```

**Comment by keshav-nangare** [comment]
Solution for the Failed tests:

- req12:

 - The test "req12" is failing due to the multiple `'ServiceTypeVersion'` element present in the response.
 - We can observe this behavior using following URL:  
               http://arcgis-ogcoregon-393577611.us-west-2.elb.amazonaws.com/arcgis/services/wcs201/GeoDataServer/WCSServer?SERVICE=WCS&VERSION=2.0.1&REQUEST=GetCapabilities 
 - We have to iterate and compare with all elements for matching version and if matching version is not found then it will produce the error message.

- req24 and req41:

 - The server returned the exception code with only one value i.e. exceptionCode="NoSuchCoverage", but test expects "status code" too.
 - The test expects two parameter in response status and exception code. Currently the server returns the only one "exceptionCode".
 - If the server returns the "status code" in the exception report (response), then this issue will resolved automatically.

Following is the result of test "req12" after change:

[test session results](https://cloud.githubusercontent.com/assets/14732650/20097792/4baa01e0-a5d6-11e6-9817-b118b99bd28d.png)

**Comment by keshav-nangare** [comment]
Fixed with this commit 7ed27e41704eca5eac4e0f0fce86ab20f9077579
